roles/encryption_configure/tasks/add_host_entry.yml

```diff
@@ -0,0 +1,13 @@
+- name: "************************ Encryption | Host Entry | GKLM Servers DNS ************************"
```

Comment:
@Jhony-Louies Can you please keep the existing format for name header . we are not using (*********) .

So first entry should be like `configure`

Reply:
Updated